src/config/firebase-config.ts

```diff
@@ -0,0 +1,12 @@
+export const config = {
+  firebaseConfig: {
+    apiKey: process.env.REACT_APP_API_KEY,
+    authDomain: process.env.REACT_APP_AUTH_DOMAIN,
+    databaseURL: process.env.REACT_APP_DATABASE_URL,
+    projectId: process.env.REACT_APP_PROJECT_ID,
+    storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
+    messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
+    appId: process.env.REACT_APP_APP_ID,
+    measurementId: process.env.REACT_APP_MEASUREMENT_ID
+  },
+};
```

Comment:
Here I've replaced the actual values by injecting them from the dotenv file, whichever one happens to be read from at execution time. This is nice if we do end up with multiple environments because we just load a different dotenv file to swap credentials around.